ee/clickhouse/views/test/test_clickhouse_action.py

```diff
@@ -257,3 +257,83 @@ def test_breakdown_by_event_property_none_people_endpoint(self):
             },
         ).json()
         self.assertEqual(len(people["results"][0]["people"]), 1)
+
+    def _test_interval(self, date_from, interval, timestamps):
```

Comment:
Note (cc @EDsCODE) - these tests mostly all fail on postgres since we don't do any special-case date handling there. Thoughts, shall I just file a bug in the backlog (want to avoid scope creep)?